### DISCUSSION TITLE
Best way to cite openMVS?

### DISCUSSION THREAD

**Discussion by For-The-Wolf** [cdcseacave/openMVS#860] [Q&A] [answered]
Do you have a preference in how openMVS should be cited in an academic publication? If you could provide a preferred bibtex entry that would be ideal. 

Previous publications seem to have just linked to the GitHub repo so I will default to this otherwise

**Comment by cdcseacave** [comment, marked as answer]
all details, including citation are in the Wiki: https://github.com/cdcseacave/openMVS/wiki

I'd like to underline to use the latest develop branch if you plan to compare to OpenMVS; these days I see all the time papers comparing to OpenMVS of 3-4 years ago, which is not only unfair, it is completely useless for everybody (except for the author of the paper, of course)

**Reply by For-The-Wolf** [reply]
Thanks! Sorry, I must have missed this.
